modules/govuk/manifests/apps/rummager.pp

```diff
@@ -67,5 +98,15 @@
     "${title}-PUBLISHING_API_BEARER_TOKEN":
       varname => 'PUBLISHING_API_BEARER_TOKEN',
       value   => $publishing_api_bearer_token;
+    'AMQP_EXCHANGE':
+      value => $rabbitmq_exchange;
+    'AMQP_MESSAGE_QUEUE':
+      value => $rabbitmq_queue;
   }
+
+  # Add this?
```

Comment:
I think this needs to be `govuk::procfile::worker`  with the `process_type` parameter set to the name of the worker (ie, the first bit of the line in the Procfile).